src/main/app/src/components/hakutulos/Hakutulos.tsx

```diff
@@ -44,17 +44,31 @@ import { MobileFiltersOnTopMenu } from './MobileFiltersOnTopMenu';
 import { Pagination } from './Pagination';
 
 const useStyles = makeStyles((theme) => ({
+  toggleWrapper: {
+    'min-width': 400,
+  },
   hakutulosSisalto: {
     maxWidth: 1600,
     margin: 'auto',
   },
+  rajaaTuloksia: {
+    [theme.breakpoints.up('sm')]: {
+      'min-width': 300,
+    },
+  },
   paperRoot: {
     width: '100%',
     boxShadow: 'none',
-    [theme.breakpoints.down('xl')]: {
+    [theme.breakpoints.up(1921)]: {
       padding: theme.spacing(1, 11),
     },
-    [theme.breakpoints.down('md')]: {
+    [theme.breakpoints.between(1601, 1921)]: {
+      padding: theme.spacing(1, 11),
+    },
+    [theme.breakpoints.between(900, 1601)]: {
+      padding: theme.spacing(1, 2),
+    },
+    [theme.breakpoints.down(900)]: {
```

Comment:
Olisi tosi hyvä, jos voitas pysyä jossain yhdenmukasissa breakpointeissa. Olen jo kerran siivonnut näitä kun oli monessa kohtaa käytetty hieman eri pikseliarvoja breakpointeille. Pitänee kustomoida noita breakpointteja vähän. Voitas katsoa tätä yhdessä.